src/webapi/tests/integ.rs

```diff
@@ -52,3 +88,78 @@ async fn ping_works() {
     assert!(response.status().is_success());
     assert_eq!(response.json::<String>().await.unwrap(), "pong");
 }
+
+#[tokio::test]
+async fn creating_user_returns_a_200_for_valid_data() {
```

Comment:
We probably don't want ALL integ tests in one file. The user tests should probably go into their own file.